lib/vm/src/instance/mod.rs

```diff
@@ -823,7 +832,13 @@ impl InstanceHandle {
             .map(|m| m.vmglobal())
             .collect::<PrimaryMap<LocalGlobalIndex, _>>()
             .into_boxed_slice();
-        let passive_data = RefCell::new(module.passive_data.clone());
+        let passive_data = RefCell::new(
+            module
+                .passive_data
+                .values()
+                .map(|data| Some(data.clone()))
```

Comment:
Consider using `.cloned()` instead, it does exactly this

Reply:
We are going from a non `Option` to an `Option` so we need to wrap in a `Some`?

Reply:
Oh I see, good call!

You can do it in two lines as `.values().cloned().map(Some)`, which is a bit more explicit but what you have is fine!